README.md

```diff
@@ -1,114 +1,85 @@
 
-Install Chrome WebDriver : The `tests/config.js` in this package is pre-configured to use Chrome WebDriver.
-* Under Debian GNU/Linux install it with `apt install chromium-driver` (Chromium is the free software version of Chrome).
-* Under Mac OS X install it with `brew install chromedriver`
 
-### Run the integration tests
+## Run the server
 
-- Terminal window 1 : run Selenium standalone server
+`cd` to the cloned `legislation-explorer` directory, and run the following commands:
 
 ```sh
-java -jar selenium-server-standalone-3.4.0.jar
-18:51:36.494 INFO - Selenium build info: version: '3.3.0', revision: 'b526bd5'
-[...]
-18:51:36.722 INFO - Selenium Server is up and running
+npm run build  # compile the frontend code
+npm start  # start the server
 ```
-- Terminal window 2 : run Chrome WebDriver
 
-```sh
-chromedriver
-Starting ChromeDriver 2.25 (undefined) on port 9515
-Only local connections are allowed.
-```
 
-- Terminal window 3 : start the application development server if not already done (`npm run dev:prod-api`).
+## Improve the app
 
-- Terminal window 4 : run the integration tests
+You can edit all files in the source folder you cloned. In order to ease development, a different set of commands will allow you to run the app with hot module replacement, which will reflect your changes almost instantly rather than rebuilding the whole package.
+
+`cd` to the cloned `legislation-explorer` directory, and run the following commands:
 
 ```sh
-npm run test:integration
-> legislation-explorer@0.0.1 test:integration /home/cbenz/Dev/openfisca/legislation-explorer
-> watai tests
-
-⨁  tests
-✔  The header should show metadata on the loaded tax benefit system.
-✔  The search should filter the list.
-✔  Navigating to an item of the list should show details, going back should keep the search.
+npm install  # install development dependencies
+npm run dev # To use http://localhost:5000/ for the Web API
+npm run dev:prod-api # To use https://api.openfisca.fr/ for the Web API
 ```
 
-## OpenFisca Documentation
+> Some additional commands can be useful for development. You can discover all of them by running `npm run`.
+
 
-Please consult http://openfisca.org/doc/
+### Tests
 
-## Contribute
+This app has both unit and integration tests. Its integration tests are implemented with [Watai](https://github.com/MattiSG/Watai). You will first need to [install its Selenium WebDriver dependencies](https://github.com/MattiSG/Watai/wiki/Installing) to run these tests.
 
-OpenFisca is a free software project.
-Its source code is distributed under the [GNU Affero General Public Licence](http://www.gnu.org/licenses/agpl.html)
-version 3 or later (see COPYING).
+Then, build the app, [run the server](#run-the-server), [run a Selenium instance](https://github.com/MattiSG/Watai/wiki/Installing#selenium-server) and, in another Terminal, run:
```

Comment:
I have an error when running Selenium. Am I the only one ?
```
events.js:137
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/Users/.../.nvm/versions/node/v9.5.0/lib/node_modules/selenium-server/lib/runner/../../logs/selenium.out'

```

Reply:
Never saw that. Probably related to using `nvm`, but I don't really feel like investigating, especially for an unstable Node version.

Reply:
Uhm, do we assume an LTS node version?